backend/app/routers/chat.py

```
@@ -346,40 +345,105 @@ def _maybe_create_alert_from_nl(nl: str, parsed: Dict[str, Any]) -> Optional[Dic
+        "actions": actions,
+        "severity": severity,
+        "cooldown_sec": cooldown_sec,
         "created_at": iso_now(),
         "updated_at": iso_now(),
     }
     try:
         ins = alerts_col.insert_one(doc)
         return {"id": str(ins.inserted_id), "name": doc["name"], "enabled": True}
-    except Exception:
+    except Exception as e:
+        logger.exception("Failed to insert NL alert `%s`: %s", doc.get("name"), e)
```

Ruff (0.15.1)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)